website/views/pages/get-started.ejs

```diff
@@ -0,0 +1,107 @@
+<div id="get-started">
+  <div class="footer-fade pb-5">
+    <div
+      style="max-width: 1260px"
+      class="container-fluid pt-5 px-4 px-sm-5 mx-auto"
+    >
+      <section id="install-section" class="py-4 mx-auto">
+        <h1>Get Started with Fleet</h1>
+        <p class="mb-4">
+          No need to put the kettle on, you'll be up and running before you can
+          say "osquery."
+        </p>
+        <img
+          alt="Fleet introduction to hosts"
+          src="/images/intro-pic-800x364@2x.png"
+          style="max-width: 900px; height: auto"
+          class="w-100 my-4 mx-auto"
+        />
+      </section>
+      <section id="try-fleet" class="py-4 mx-auto">
+        <h1>Try Fleet</h1>
```

Comment:
```suggestion
        <h2 class="mb-3">Try Fleet</h2>
```
I believe search engines like one `<h1>` tag per document. I could be wrong about this.

I also suggest adding some `margin-bottom`.